share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
> Would you be OK with asking people to not file issues and/or saying that they are unsupported?

Would such a note only apply to the shellvar? The flag `--build-from-source` shouldn't be unsupported.